admin/code/messages/InvalidBatchAction.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+return function(array $action) {
+	$action = trim(print_r($action, true));
+
+	return "
```

Comment:
I remove the previous `preg_replace` from here - print_r was just destroying it.